src/renderer/store/modules/camera/camera.mutations.js

```diff
@@ -0,0 +1,18 @@
+import {
+  SET_INTERVAL,
+  SET_CURRENT_FRAME,
+  SET_LOAD_IMAGES_SAMPLE,
+  SET_IMAGES_SAMPLE,
+  PUSH_IMAGE_TO_IMAGES_SAMPLE
+} from './mutation-types'
+
+export default {
+  [SET_INTERVAL]: (state, interval) => (state.cameraInterval = interval), // eslint-disable-line
```

Comment:
Seems to be the same problem with the linter as earlier. Maybe we should change the rule ?